test/spec/modules/33acrossIdSystem_spec.js

```diff
@@ -157,6 +157,48 @@ describe('33acrossIdSystem', () => {
       });
     });
 
+    context('when coppa is true', () => {
+      it('should call endpoint with the coppa=1', () => {
+        const completeCallback = () => {};
+        const { callback } = thirthyThreeAcrossIdSubmodule.getId({
+          params: {
+            pid: '12345'
+          }
+        });
+
+        sinon.stub(coppaDataHandler, 'getCoppa').returns(true);
+
+        callback(completeCallback);
+
+        const [request] = server.requests;
+
+        expect(request.url).to.contain('coppa=1');
+
+        coppaDataHandler.getCoppa.restore();
+      });
+    });
+
+    context('when coppa is false', () => {
```

Comment:
here we don't have to check for an `undefined` coppa, since `getCoppa` always returns a boolean.